CMake fails to run

Build log:

```
~snip~
-- Try C++11 flag = [-std=gnu++0x]
-- Performing Test CXX11_FLAG_DETECTED
-- Performing Test CXX11_FLAG_DETECTED - Success
-- Found clang: /home/jill/build/git/wclang/clang-4.0
CMake Error at CMakeLists.txt:71 (string):
  string sub-command REGEX, mode REPLACE needs at least 6 arguments total to
  command.


-- Found mingw-gcc: /usr/bin/i686-w64-mingw32-gcc-win32
-- Found mingw-gcc: /usr/bin/x86_64-w64-mingw32-gcc-win32
-- Configuring incomplete, errors occurred!
~snip~
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,7 +70,7 @@ execute_process ( COMMAND ${CLANG_C_COMPILER} --version
                 )
 string (REGEX REPLACE ".*version ([0-9\\.]+).*"
          "\\1" CLANG_VERSION
-         ${CLANG_VERSION})
+         "${CLANG_VERSION}")
 if (CLANG_VERSION VERSION_GREATER 3.0)
   set (CLANG_TARGET_OPT_FLAG -target)
 else ()

